### DISCUSSION TITLE
BelongsToSelect create action /

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#2333] [Help]
how can in implement a  create action in a BelongsToSelect Field?

**Comment by danharrin** [comment]
https://filamentphp.com/docs/2.x/forms/fields#creating-new-records

**Reply by BenQoder** [reply]
I do not think this works correctly.

I have created the form as instructed on the documentation but when I click the plus (+) sign, the modal doesn't come up at all as shown on the twitter demo.

**Reply by BenQoder** [reply]
When I check network requests, it sends requests to Livewire but no ui changes at all.

I will really appreciate any feedback.. Thanks

**Reply by danharrin** [reply]
If you're using the form builder outside the admin panel, are you also using `{{ $this->modal }}` in the view?

**Reply by BenQoder** [reply]
I am using the it inside Admin Panel

**Reply by BenQoder** [reply]
I saw in another comment where asked if filament view files have been customised... 

That was my problem.. thank you for your time.

**Reply by danharrin** [reply]
Yeah, definitely delete any overriden views that you dont need. Use CSS where possible!
